documentation/book/oauth/proc-oauth-server-config.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// assembly-oauth-strimzi-config.adoc
+
+[id='proc-oauth-server-config-{context}']
+= Deploying an {oauth} authorization server
+
+This procedure describes how to deploy an authorization server and configure it for integration with {ProductName}.
```

Comment:
So, I wonder ow much this works with something else than Keycloak. In either case we should probably not use the word `deploy` here and in the chapter name. It suggests you will show someone how to deploy it. But this is just a generic configuration description. The deployment has to be done by the user on his/her own (which is fine, we should just not call it deploying).

Reply:
Okay. I've changed the title and introduced the chapter with: 'This procedure describes how to configure an authorization server for integration with {ProductName}.'

This procedure is work in progress. I've asked @mstruk for help in working out a set of generic instructions.

Reply:
This procedure has changed a lot since the review by @mstruck. Now simpler.